modAionImpl/src/org/aion/zero/impl/sync/msg/ResStatus.java

```diff
@@ -9,7 +9,7 @@
 /** @author Chris */
 public final class ResStatus extends Msg {
 
-    private static final int minLen = 8 + 1 + 1 + 32 + 32;
+    private static final int minLen = 8 + 1 + 1 + 32 + 32 + 1 + 2 + 1 + 1 + 4;
```

Comment:
need to consider the old protocol compatibilities. Means the old kernel can also be decoded correctly